### PR TITLE
Userinputtoapi

### PR DESCRIPTION
**Pull request by Jason-Peralta**
user input handling for accurate api url

### CODE REVIEW

File: src/components/apibutton.js [hunk 1]
```diff
@@ -2,11 +2,93 @@ import React, {useEffect} from 'react';
...
+                                :(stateURL === 'New York City')
+                                    ? "KNYC"
+                                    :(stateURL === 'Philadelphia')
+                                        ? "KPHL"
+                                        :(stateURL === 'Phoenix')
+                                            ? "KPHX"
+                                            :(stateURL === 'Seattle')
+                                                ? "KSEA"
+                                                : ""
+    }
```
Comment (alachhman) [inline src/components/apibutton.js]: Turn this into a switch case

File: src/components/statefilter.js [hunk 2]
```diff
@@ -8,9 +8,9 @@ export default function StateFilter(props) {
     return (
         <Pane display="flex" alignItems="center" padding={8} marginX={majorScale(3)}>
             <SelectMenu
-                title="Select State"
+                title="Select City"
                 onFilterChange={(filter) => setFilter(filter)}
-                options={['New York', 'New Jersey'].map((label) => ({ label, value: label }))}
+                options={['Charlotte', 'Los Angeles','Houston', 'Indianapolis', 'Jacksonville','Chicago', 'New York City', 'Philadelphia','Phoenix', 'Seattle'].map((label) => ({ label, value: label }))}
```
Comment (alachhman) [inline src/components/statefilter.js]: Abstract the options array to a variable outside the scope of this component. Can keep the map() on this line exactly the same

**Review by alachhman — CHANGES_REQUESTED** [review]

File: src/components/statefilter.js (hunk 2 above)
Comment (Jason-Peralta) [reply src/components/statefilter.js]: changed

File: src/components/apibutton.js (hunk 1 above)
Comment (Jason-Peralta) [reply src/components/apibutton.js]: changed

**Review by alachhman — APPROVED** [review]
LGTM

**Pull request merged**